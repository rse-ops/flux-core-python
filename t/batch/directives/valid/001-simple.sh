#!/bin/sh
# simple directives
#FLUX: -N4                  # Request four nodes
#FLUX: --queue=batch        # Submit to the batch queue
#FLUX: --job-name=app001    # Set an explicit job name
flux mini run -N4 app
