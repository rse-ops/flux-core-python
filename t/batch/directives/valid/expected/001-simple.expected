SETARGS(['-N4'])
SETARGS(['--queue=batch'])
SETARGS(['--job-name=app001'])
